src/circuit/gadget/ecc/chip/add.rs

```diff
@@ -210,17 +207,17 @@ pub(super) fn assign_region<C: CurveAffine>(
                     region.assign_advice(|| "set lambda", lambda, offset, || Ok(lambda_val))?;
                 }
 
-                if x_p == C::Base::zero() {
+                if x_p == F::zero() {
```

Comment:
Similarly we don't assign lambda in these cases. Will this cause a problem?

Reply:
Resolving this comment since `add.rs` has undergone significant fixes.